app/models/patient.rb

```diff
@@ -624,6 +624,12 @@ def report_eligibility
       messages << { message: 'Monitoree was purged', datetime: nil }
     end
 
+    # Can't send messages to monitorees that are on the closed line list.
+    unless monitoring
```

Comment:
The reason I never included this in the first place is if dependents are present, it changes the fact that this person may indeed get notifications. I think that just means this needs to have slightly more involved logic (rather than just "monitoring"). Thoughts?

Reply:
Woof, I must be tired. Yeah of course that definitely needs to be included. Ultimately, I'd like to not have to repeat the logic here that we already sort of do in the `reminder_eligible` scope and `send_assessment` method since this is just adding more places we need to update whenever there's a change. I think once we break out the logic from send_assessment we should refactor this as well. But that's for the future. I'll get in a change so that this checks for active dependents.

Reply:
Done.